### PR TITLE
Add methods to FilesUtil

### PR DESCRIPTION
**Pull request by daniel-ohayon**
Depends on 
* https://github.com/genestack/genestack/pull/2077 (PR for JAVA `getMetainfoValuesAsStrings`)
* https://github.com/genestack/genestack/pull/2086 (PR for JAVA `linkFiles` / `unlinkFiles`)

### CODE REVIEW

**Comment by daniel-ohayon** [comment]
Should we deprecate `link_file` and `unlink_file` ?

File: genestack_client/files_util.py [hunk 1]
```diff
@@ -194,6 +194,32 @@ def unlink_file(self, accession, parent):
         """
         self.invoke('unlinkFile', accession, parent)
 
+    def link_files(self, accessions, parents):
```
Comment (genestack-solomatin) [inline genestack_client/files_util.py]: I prefer to have collection of pairs
```python
[(parent1, accession1), (parent1, accession2)]
```
Or more complex structure there each first name is parent and other are childs
```python
(parent1, accession1, accession2, accession3), (parent2, accession4, accession5)
```

**Comment by genestack-solomatin** [comment]
This looks like optimization `link_files` and `unlink_files` how many time you save with them?

**Comment by daniel-ohayon** [comment]
Yes. I wrote the following benchmark script:
```python
from genestack_client import *
import time

TEST_FOLDER = "GSF1514929"
connection = get_connection()
files_util = FilesUtil(connection)
children = files_util.get_file_children(TEST_FOLDER)
temp_folder = files_util.create_folder("Temp")
temp_folder2 = files_util.create_folder("Temp2")
print "Linking %d files into a single folder" % len(children) 
print "Using link_file:"
time1 = time.time()
for child in children:
    files_util.link_file(child, temp_folder)
print "Elapsed: %d seconds" % (time.time() - time1)

print "Using link_files:"
time2 = time.time()
files_util.invoke('linkFiles',children, [temp_folder2 for _ in range(len(children))])
print "Elapsed: %d seconds" % (time.time() - time2)
```

And the results I get on `int-dev` 
```
Linking 48 files into a single folder
Using link_file:
Elapsed: 25 seconds
Using link_files:
Elapsed: 2 seconds
```

File: genestack_client/files_util.py (hunk 1 above)
Comment (daniel-ohayon) [inline genestack_client/files_util.py]: I agree that the structure isn't very nice. How about this: a `Map<String,List<String>>` like 
```
{
   parent1: [child1, child2]
   parent2: [child1]
   ...
}
```

Comment (genestack-solomatin) [inline genestack_client/files_util.py]: Map looks nice. Lets implement it

**Comment by genestack-solomatin** [comment]
Please, rewrite `[un]link_file` to use this new methods.

**Pull request merged**